File: SwappingQuestion/Swapping.js
let num1 = parseInt(prompt("enter the first number"));
let num2 = parseInt(prompt("enter the second number"));
console.log("before the swapping",num1);
console.log("before the swapping",num2);
num1 = num1+num2;
num2 = num1-num2;
num1 = num1-num2;
console.log("after the swapping",num1);
console.log("after the swapping",num2);

File: SwappingQuestion/Swapping.rb
name1 = prompt"enter the number"
    name1 = gets.chomp
    name2 = prompt"enter the second name"
    name2 = gets.chomp
    puts"before the swapping" #{name1}
    puts"before the swapping" #{name2}
    name1,name2 = name2,name1
    puts"after the swapping" #{name1}
    puts"after the swapping" #{name2}
    
    